src/main/java/org/dependencytrack/parser/osv/model/Ecosystem.java

```diff
@@ -17,7 +17,19 @@ public enum Ecosystem {
     UVI("UVI"),
     RUST("crates.io"),
     NPM("npm"),
-    DWF("DWF");
+    DWF("DWF"),
+    DEBIAN("Debian"),
+    DEBIAN_11("Debian:11"),
+    DEBIAN_10("Debian:10"),
+    DEBIAN_9("Debian:9"),
+    DEBIAN_8("Debian:8"),
+    DEBIAN_7("Debian:7"),
+    DEBIAN_6("Debian:6.0"),
+    DEBIAN_5("Debian:5.0"),
+    DEBIAN_4("Debian:4.0"),
+    DEBIAN_3_1("Debian:3.1"),
+    DEBIAN_3_0("Debian:3.0"),
+    GITHUB_ACTIONS("GitHub Actions");
```

Comment:
The `value`s in this enum are inserted as-is into a URL string in `OsvDownloadTask`. The space here is thus not properly escaped and causes an exception, e.g.:

```
2022-08-05 18:59:19,866 INFO [OsvDownloadTask] Updating datasource with Google OSV advisories for ecosystem GitHub Actions
2022-08-05 18:59:19,870 ERROR [LoggableUncaughtExceptionHandler] An unknown error occurred in an asynchronous event or notification thread
java.lang.IllegalArgumentException: Illegal character in path at index 57: https://osv-vulnerabilities.storage.googleapis.com/GitHub Actions/all.zip
	at java.base/java.net.URI.create(URI.java:883)
	at org.apache.http.client.methods.HttpGet.<init>(HttpGet.java:66)
	at org.dependencytrack.tasks.OsvDownloadTask.inform(OsvDownloadTask.java:67)
	at alpine.event.framework.BaseEventService.lambda$publish$0(BaseEventService.java:101)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

We should add escaping to `OsvDownloadTask` to prevent this.